data/data_test.go

```diff
@@ -364,8 +364,7 @@ func TestData_DelFeed(t *testing.T) {
 	})
 }
 
-func testRootError(t *testing.T, re *RootError, rp *RootPack,
-	pk cipher.PubKey) {
+func testRootError(t *testing.T, re *RootError, rp *RootPack,	pk cipher.PubKey) {
```

Comment:
lol seriously? I have split screen with two textmate windows and it still wider then 80 symbols :)